Sources/Sentry/SentryANRTracker.m

```diff
@@ -6,47 +6,42 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-#if SENTRY_HAS_UIKIT
-
 @interface
 SentryANRTracker ()
 
-@property (weak, nonatomic) id<SentryANRTrackerDelegate> delegate;
-@property (nonatomic, assign) NSTimeInterval timeoutInterval;
 @property (nonatomic, strong) id<SentryCurrentDateProvider> currentDate;
 @property (nonatomic, strong) SentryCrashWrapper *crashWrapper;
 @property (nonatomic, strong) SentryDispatchQueueWrapper *dispatchQueueWrapper;
 @property (nonatomic, strong) SentryThreadWrapper *threadWrapper;
+@property (nonatomic, strong) NSMutableSet<id<SentryANRTrackerDelegate>> *listeners;
```

Comment:
`h`: Doesn't this create a retain cycle? Previously, the delegate was `weak`, now we have strong pointers. Maybe we have to use a `NSPointerArray` with `NSPointerFunctionsWeakMemory`?

Reply:
Yes makes sense, and they remove themselves in `uninstall`.